### ISSUE TITLE
[Web] Starting debug with VSCode breaks after a while

### ISSUE THREAD

**Issue by apadol-adeo** [Dart-Code/Dart-Code#3518]
Hello,

I am facing an issue that I think is linked to the VSCode extension for flutter.
I have a project that runs fine using this VSCode configuration:
```json
{
            "name": "web-myproject",
            "request": "launch",
            "flutterMode": "debug",
            "type": "dart",
            "args": [
                "-d",
                "chrome",
                "--web-port=5000",
            ],
            "program" : "lib/main.dart",
        },
```
What I am seeing, it's that after a few launches (about 4 or 5), the debug session does not start anymore, and when I try to start a new debug session I end up with a broken chrome window like this:

![Screenshot 2021-08-05 at 17 49 41](https://user-images.githubusercontent.com/84775091/128474488-b49b229d-62ea-43dc-b3ed-96b9abf58d9e.png)

Whenever I am presented with this broken chrome window, if I start the application using the command line:

`flutter run -d chrome --web-port=5000`

everything works as expected.

The only thing that restores the behavior of the VSCode extension, is to reboot my mac. Rebooting VSCode and/or cleaning the flutter project does not help.

I attached the dart logs when reproducing the issue.
[Dart-Code-Log-2021-07-04.17-43-54.txt](https://github.com/Dart-Code/Dart-Code/files/6943962/Dart-Code-Log-2021-07-04.17-43-54.txt)

Any help would be appreciated.

Thanks

[✓] Flutter (Channel stable, 2.2.3, on macOS 11.4 20F71 darwin-x64, locale
    en-GB)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1)
[✓] VS Code (version 1.59.0)
Flutter VSCode extension (v3.25.0)
Dart VSCode extension (v3.25.1)

**Comment by DanTup** [comment]
@annagrin does this seem like a webdev issue? There's nothing obviously strange in the logs, but the last few lines include sending breakpoints, which seems a little similar to https://github.com/dart-lang/webdev/issues/1271. Based on the date I would expect that fix was included v2.2.3 of Flutter (though I'm not certain).

**Comment by DanTup** [comment]
Looks like @annagrin fixed this in https://github.com/dart-lang/webdev/pull/1384. Thanks!
